src/Resources/app/administration/src/sw-order-detail-override/sw-order-detail.html.twig

```diff
@@ -0,0 +1,7 @@
+{% block sw_order_detail_header_title %}
+    {% parent %}
+{% endblock %}
```

Comment:
I believe this isn't needed since it is only calling the parent?

Reply:
True. I'll take it out and do one more test